aiokafka/producer.py

```diff
@@ -240,7 +287,8 @@ def send(self, topic, value=None, key=None, partition=None):
                 key_serializer.
 
         Returns:
-            response from Kafka
+            asyncio.Future: future object that will be set when message is
```

Comment:
I do see some sense to do it though. The first may wait till we actually add object to `send` buffer.